### ISSUE TITLE
Multiple devices problem

### ISSUE THREAD

**Issue by Philipboy881** [ChrisHaPunkt/ha-stihl-imow#10]
I have 5 mowers and it only shows the first mower as listed in the imow app, is it possible to add support for multiple mowers ?

**Comment by ChrisHaPunkt** [comment]
It is possible. Sadly, I currently do not have the time to implement this (we just got our 2nd child ;) ). But feel free to contribute some python code ;)

**Comment by klaaspiet** [comment]
Yes it is possible.  i have copilot make a code. An the first try it is working in HA.  i have 2 mowers now in my HA.

@Philipboy881 do you still need this

**Comment by ChrisHaPunkt** [comment]
Version 2.0.0 should provide this functionality.
